public/main.js

```diff
@@ -8,6 +8,11 @@ var load = function () {
     });
 }
 
+var editTask = function(name, done){
```

Comment:
Editar task com mesmo nome teria problemas

Reply:
Mas como pegar o valor do `Id` da task aqui? O `name` que tava usando é um parâmetro da function.
Se for fazer uma consulta no banco e houver `name` duplicados vai dar no mesmo problema.